sirepo/http_reply.py

```diff
@@ -75,8 +75,8 @@ def f():
         if isinstance(content_or_path, pkconst.PY_PATH_LOCAL_TYPE):
             return flask.send_file(str(content_or_path))
         if content_type == 'application/json':
-            return flask.current_app.response_class(pkjson.dump_pretty(content_or_path))
-        return flask.current_app.response_class(content_or_path)
+            return sirepo.util.flask_app().response_class(pkjson.dump_pretty(content_or_path))
```

Comment:
Add and use liberally:

```py
def gen_response(*args, **kwargs):
    return sirepo.util.flask_app().response_class(*args, **kwargs)
```

There are some places that call `flask.Response` directly, which should be changed to this. Then we have one more encapsulation to allow migration to tornado without the need to remove "flask" from N places.